osbenchmark/results_publisher.py

```diff
@@ -464,16 +464,16 @@ def _write_results(self, metrics_table, metrics_table_console):
                             data_plain=metrics_table, data_rich=metrics_table_console)
 
     def _publish_throughput(self, baseline_stats, contender_stats, task):
-        b_min = baseline_stats.metrics(task)["throughput"]["min"]
+        b_min = baseline_stats.metrics(task)["throughput"].get("overall_min") or baseline_stats.metrics(task)["throughput"]["min"]
```

Comment:
It would probably be better to set the `overall_min` key prior, so dealing with a special case won't be necessary.

Reply:
I think this logic is necessary since the results publisher is also used for normal test executions which will not contain 'overall' min/max values